Sources/StreamChat/Controllers/MessageController/MessageController.swift

```diff
@@ -536,6 +536,21 @@ public extension ChatMessageController {
             }
         }
     }
+    
+    /// Translates the message to the given language.
+    /// The translated message will be returned via `didChangeMessage` delegate callback.
+    /// Translation will be in `message.translations[language]`
+    /// - Parameters:
+    ///   - language: The language message text should be translated to.
+    ///   - completion: The completion. Will be called on a **callbackQueue** when the operation is finished.
+    ///                 If operation fails, the completion is called with the error.
+    func translate(to language: TranslationLanguage, completion: ((Error?) -> Void)? = nil) {
```

Comment:
Can it be valuable for a customer to be able to translate message to multiple language with a single run? 
```swift
func translate(to languages: [TranslationLanguage], completion: ((Error?) -> Void)? = nil) { 
 ...
}

func translate(to language: TranslationLanguage, completion: ((Error?) -> Void)? = nil) {
 translate(to: [language], completion: completion)
} 
```

Reply:
All SDKs have this type of API, there's no need it for this now. Backend also doesn't accept more than 1 language, it'd be N requests wrapped into 1 call